android/src/main/java/com/optimove/flutter/OptimoveInitProvider.java

```diff
@@ -41,6 +42,11 @@ public class OptimoveInitProvider extends ContentProvider {
     private static final String IN_APP_EXPLICIT_BY_USER_KEY = "explicit-by-user";
     private static final String ENABLE_DDL_KEY = "enableDeferredDeepLinking";
 
+    private static final String SDK_VERSION = "2.0.0";
+    private static final int SDK_TYPE = 105;
+    private static final int RUNTIME_TYPE = 9;
+    private static final String RUNTIME_VERSION = "3.0.1";
```

Comment:
always 3.0.1?

Reply:
i think if not possible to get correct version, it's fine to send 'Unknown'

Reply:
Changed to unknown